spec/rails_helper.rb

```diff
@@ -113,7 +113,7 @@
     stub_request(:any, /res.cloudinary.com/).to_rack("dsdsdsds")
 
     stub_request(:post, /api.fastly.com/).
-      to_return(status: 200, body: "", headers: {})
+      to_return(status: 200, body: "".to_json, headers: {})
```

Comment:
This was painful to find 😂 the `fastly` gem tries to parse the response so `JSON.parse("")` was throwing an error.